src/core/api.ts

```diff
@@ -99,7 +99,7 @@ export function createApi(
             const parsed = parseGenomicPosition(position);
             const { chromosome: chr } = parsed;
             let { start, end } = parsed;
-            if (!chr || chromInfo.interval[chr]) {
+            if (!chr || !chromInfo.interval[chr]) {
```

Comment:
I think this was a bug

Reply:
I wonder if we can set up tests for the APIs not to break things.

Reply:
then the zoomTo implementation would turn into:

```typescript

function zoomTo(viewId: string, positionString: string, padding = 0, duration = 1000) {
  const assembly = getTrack(viewId)?.spec.assembly;
  const position = GenomicPosition.fromString(positionString);
  let coordinates: [number, number];
  try {
    coordinates = position.toAbsoluteCoordinates(assembly);
  } catch (e) {
     console.warn("chromosome is not valid")
     return;
  }
  hg.api.zoomTo(viewId, ...coordinates, ...coordinates, duration);
}

```

Reply:
and we could test the `GenomicPosition` class for a variety of positions and assemblies and assert the correct behavior

Reply:
I've updated the codes accordingly in [the last commit](https://github.com/gosling-lang/gosling.js/pull/796/commits/846da78d83480c44ac56aad687db1f90cd596d7f). I think we can expand this class and reuse it throughout the codes.